src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -414,8 +411,8 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
           ) : (
             <>
               {this.renderEditionSets(
-                artworkEcommerceAvailable,
-                editionSelectableOnInquireable
+                artworkEcommerceAvailable || !!isInquireable,
+                !!isInquireable
```

Comment:
Why are we checking twice if `!!isInquireable` , is the first one strictly related to `artworkEcommerceAvailable` and doesn't work with just `!!isInquireable`?

Reply:
looks like a checking but I only kept the same behavior to the `renderEditionSets` function, which expected the first param to also check `isInquireable`, the second parameter was also `isInquireable`. I can refactor the function if you ask 😬

Reply:
I refactored a bit to pick artwork prop inside the renderEditionSet function in [dc6953f](https://github.com/artsy/force/pull/9846/commits/dc6953f74d5bef6419467ecd9879483dd00720b3) 👍